### PR TITLE
Add fetch flags

### PR DESCRIPTION
**Pull request by naviechan**
This PR is in conjunction with [#7417](https://github.com/ledgerwatch/erigon/pull/7417) in Erigon.

This creates `Fetch Flags` button and `ui/flags` in `ui_handler.go` to fetch context flags via `/debug/metrics/flags` in Erigon.

### CODE REVIEW

**Comment by AlexeyAkhunov** [comment]
Thank you! Looks good. Could you also add 2 more things:
1) section in the documentation `README.md` about this new `Fetch Flags` functionality, similar to what is done for other functions
2) Once the `Version` is incremented in Erigon's `diagnostics/version.go`, the diagnostics system may be dealing with version of Erigon node which have `Version = 1` and `Version = 2`. Could you make it so that `Fetch Flags` only works for `Version >= 2` and if the version is too small, display message about it

**Comment by naviechan** [comment]
Regarding your second point, I think this is in the area of API versioning which is tricky and requires some more thought. 

For now I have implemented a temporary solution and come up with a better solution later

**Comment by naviechan** [comment]
> Thank you! Looks good. Could you also add 2 more things:
> 
> 1. section in the documentation `README.md` about this new `Fetch Flags` functionality, similar to what is done for other functions
> 2. Once the `Version` is incremented in Erigon's `diagnostics/version.go`, the diagnostics system may be dealing with version of Erigon node which have `Version = 1` and `Version = 2`. Could you make it so that `Fetch Flags` only works for `Version >= 2` and if the version is too small, display message about it

Regarding your second point, I think this is in the area of API versioning which is tricky and requires some more thought. 

For now I have implemented a temporary solution and come up with a better solution later

File: cmd/flags.go [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+				}
+			}
+		} else {
+			flags.Error = fmt.Sprintf("incorrect response (first line needs to be SUCCESS): %v", lines)
+		}
+	} else {
+		flags.Error = result
+	}
+	if err := templ.ExecuteTemplate(w, "flags.html", flags); err != nil {
+		fmt.Fprintf(w, "Executing versions template: %v", err)
```
Comment (AlexeyAkhunov) [inline cmd/flags.go:53]: Needs to say `flags template`

File: cmd/flags.go [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+
+func processFlags(w http.ResponseWriter, templ *template.Template, success bool, result string, versions Versions) {
+
+	if !versions.Success {
+		fmt.Fprintf(w, "Unable to process flag due to inability to get node version: %s", versions.Error)
+		return
+	}
+
+	if versions.NodeVersion < 2 {
+		fmt.Fprintf(w, "Flags only support version >= 2. Node version: %d", versions.NodeVersion)
```
Comment (AlexeyAkhunov) [inline cmd/flags.go:24]: That's probably good enough for now, we shall test it

**Comment by AlexeyAkhunov** [comment]
We can have your temporary solution and add a next improvement idea to what you come up with (I am curious about your idea of how to make it better)

**Pull request merged**